Update accessibility statement for WCAG 2.2 (#250)
alphagov/notifications-tech-docs: source/accessibility.html.md

This website is run by the GOV.UK Notify team at the Government Digital Service (GDS). It is designed to be used by as many people as possible. You should be able to:

- change colours, contrast levels and fonts using browser or device settings
- zoom in up to 400% without the text spilling off the screen
- navigate most of the website using a keyboard or speech recognition software
- listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)

We’ve also made the website text as simple as possible to understand.

[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.

## How accessible this website is

We know this website is not fully accessible:

> - it has links whose only content is an image with no alt text
> - the table of contents is sometimes announced as a dialog to screen readers when it isn't
> - links can be hidden by the 'sticky' table of contents button while focused

Reviewer: ```suggestion
- it link to images that have no alt text
- screen readers sometimes incorrectly announce the table of contents as a dialog 
- links can be hidden by the 'sticky' table of contents button while focused
```

#### Non-compliance with the accessibility regulations

- the content of some links is an image with no alt text which provides no accessible name to assistive technologies. This fails
  [WCAG 2.2 success criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)
  and [WCAG 2.2 success
  criterion 2.4.4: Link Purpose (In
  Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)
- the table of contents will be announced as a dialog to screen readers but only behaves like one on smaller
  screens or at high zoom. This fails [WCAG 2.2 success criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)

> - focused links can be overlaped by the 'sticky' table of contents button which hides them from

Reviewer: ```suggestion
- focused links can be overlaped by the ‘sticky’ table of contents button which hides them from
```

  sight when they're being used. This fails [WCAG 2.2 success criterion 2.4.11: Focus Not Obscured](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html)

## How we tested this website

We last tested this website for accessibility issues in August 2024. We followed the guidance for
each of [the success criteria added in WCAG 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/) and tested the website against them.

> We first tested this website for accessibility issues in August 2020.

Reviewer: ```suggestion
```

> On that occassion, we used manual and automated tests to look for issues, following [guidance on testing for accessibility from the GOV.UK Service Manual](https://www.gov.uk/service-manual/helping-people-to-use-your-service/testing-for-accessibility).

Reviewer: ```suggestion
When we first tested this website for accessibility issues in August 2020, we used manual and automated tests to look for issues, following [guidance on testing for accessibility from the GOV.UK Service Manual](https://www.gov.uk/service-manual/helping-people-to-use-your-service/testing-for-accessibility).
```
